packages/streamline/src/components/cards/card-carousel/card-carousel.types.ts

```diff
@@ -0,0 +1,54 @@
+import { CardHeaderProps } from '../../../primitives/card/card-header/card-header.types';
+import { CardProps } from '../../../primitives/card/card.types';
+import { IconsName, Size } from '../../../primitives/icon/icon.types';
+
+type DefaultProps = Pick<
+  CardProps,
+  'accessibilityLabel' | 'testID' | 'onPress' | 'onPressIn' | 'onPressOut'
+> &
+  Omit<CardHeaderProps, 'colors' | 'title' | 'iconName'> & {
+    size: Extract<Size, 'small' | 'large'>;
+    /**
+     * Indicate the loading state of the card
+     */
+    isLoading?: boolean;
+    /**
+     * Card's title
+     */
+    title?: string;
+    /**
+     * Card's description
+     */
+    description?: string;
+    /**
+     * Card's media background
+     */
+    media?: string;
```

Comment:
Please use `ImageSourcePropType` instead of `string` so we can pass both `uri` and local images (`require('image')`)

Reply:
Fixed